lisa/tools/sockperf.py

```diff
@@ -65,8 +65,18 @@ def can_install(self) -> bool:
         r"=\s+(?P<min_latency_us>[0-9]+\.[0-9]+)"
     )
 
-    # ====> avg-rtt=3770.433 (std-dev=4585.840, mean-ad=4427.419, median-ad=171.194, siqr=4674.992, cv=1.216, std-error=369.538, 99.0% ci=[2818.566, 4722.300]) # noqa: E501
-    sockperf_average_latency = re.compile(r"avg-rtt=(?P<avg_latency_us>[0-9]+\.[0-9]+)")
+    # Summary: Round trip is 297.328 usec
+    sockperf_average_latency = re.compile(
+        r"Summary: Round trip is (?P<avg_latency_us>[0-9]+\.[0-9]+) usec"
+    )  # noqa: E501
+
+    # Total 1283 observations;
+    sockperf_total_observations = re.compile(
+        r"Total (?P<total_observations>[0-9]+) observations"
+    )  # noqa: E501
+
+    # [Valid Duration] RunTime=0.546 sec; SentMessages=1283; ReceivedMessages=1283
+    sockperf_run_time = re.compile(r"RunTime=(?P<run_time>[0-9]+\.[0-9]+) sec")
```

Comment:
will the units always be usec / sec?

Reply:
It will always be seconds based on the runs i did